isis/src/base/apps/ckwriter/CkSpiceSegment.cpp

```diff
@@ -222,12 +222,9 @@ void CkSpiceSegment::import(Cube &cube, const QString &tblname) {
     _endTime = _times[size(_times)-1];
 
     // Load necessary kernels (IAK for Cassini, mainly)
-    _kernels.Load("FK,SCLK,LSK,IAK");
-
-//    _kernels.Load("FK,SCLK,LSK,IAK,SPK");
-//   QStringList loaded = _kernels.getLoadedList();
-//   cout << "\nKernels prior to conversion...\n" << loaded.join("\n");
+    _kernels.Load("CK,FK,SCLK,LSK,IAK");
 
+    QStringList loaded = _kernels.getLoadedList();
```

Comment:
Where is this used?

Reply:
It gets used in a commented out print, I'll remove the variable